### ISSUE TITLE
Would you like to join the Circadiaware collective? :-)

### ISSUE THREAD

**Issue by lrq3000** [barrettfdavis/online_actogram#3]
Dear Barrett,

Progress has been slower than planned due to unforeseen complications (not unrelated to the current pandemic!), but is still happening :-)

I would like to invite you to the Circadiaware collective. I think we already discussed a bit about it, but now it exists since earlier this year as a GitHub organization, and will soon have its own website as a public interface to publicize our works. Here is the github repo, with already quite some projects in active development:

https://github.com/Circadiaware

I will now present what is Circadiaware :-) Circadiaware is a collective of developers of open-source tools (software and hardware) to improve the detection and management of non-24 and other circadian rhythm disorders. It's all volunteering work and all the tools will always be open-source and free. The goal of making this collective is to centralize the tools and resources, to reduce redundancy in our works (ie, reusing others libraries instead of remaking the same work again and again), and to have a more powerful platform to publicize our works and raise public awareness.

Your tool is IMHO a key element that may become a mass screening tool for circadian rhythm disorders. Here is the plan of the features me and other devs plan to contribute:

* Integration of multiple browsers' histories from any platform. The computer software will be the central software, with a GUI to simply enter all the history files, which can be exported in each respective browser. The software can of course automatically fetch the history files on the current computer, but allowing manual files input allows to include histories from other computers and even phones and tablets, which usually do not allow 3rd-party softwares to access histories and parameters as a security measure.
* Automatic detection of circadian rhythm disorders patterns. Currently, I have an algorithm that should allow to detect non-24, DSPD, ASPD and irregular sleep-wake pattern. Of course, this is not intended (and won't be stated) as a formal diagnosis, but it can provide a quick idea and some resources links for the user to further investigate if they need to seek a diagnosis and treatment. Not unlike online self-assessment scales for various neuroatypicalism disorders.
* At a later development stage, we could integrate an option for the users to allow to send their anonymized data, as to build a database of actogram patterns for various circadian rhythm disorders. This will require an Ethics Committee approval of course, so this cannot be done now, but if this gets done, this can further help in assessing the reliability of this tool (maybe up to a publication, so that other scientists could use it too ;-) ).

What joining Circadiaware entails for you and the project?
* In practice, the project will be forked. It will always stay linked to your original repo as the parent, unless you choose not to.
* You remain the first author of the project. You can gain access as the co-main admin of the forked repo too if you want.
* At least one other member of Circadiaware will also be a co-admin of the forked repo. This is to be more future proof, to always have at least 2 active admins for each repo, so that if one leaves, the other can still maintain the project or find a replacement.
* Hence, you must agree then that the project can be maintained by Circadiaware's other members. But you can choose to lead the project's development or not, up to you :-)
* The project must remain under an opensource licence. It will always be free and never be sold by Circadiaware (but of course since it's opensource it can be sold by others, we can't control that, but like any opensource project).
* The project may be renamed for marketing purposes (in the sense of brand recognition, eg, the wearable circadian monitoring tools were renamed from SleepNon24BiologicalTools to [Wearadian](https://github.com/Circadiaware/wearadian), which got featured in the [Adafruit blog](https://twitter.com/adafruit/status/1397492969649868800) - so the rebranding worked lol).
* You agree that Circadiaware can publicize the tool, with the aim of improving the detection or management of non-24 or other circadian rhythm disorders.
* There may be scientific papers published about some tools to allow scientists to use the tools. You will of course be contacted before to be invited to provide your thoughts on the paper and whether you agree to be a co-author of it. Essentially, the tools will likely get published in JOSS once they are mature enough for use in research settings (ie, we have data to prove they are useful), and co-authoring will be agreed on by everyone, but usually it's a factor of quantity and quality of contributions, with the first software author (you in this case) having a special place because of innovation.

TL;DR, joining Circadiaware means that you allow the collective to continue working on your project, which will remain free and opensource, and you can lead the development if you want to :-)

We have several developers beside me who are currently volunteering to continue working on this project :D

So, Barrett, would you like to join the Circadiaware collective and allow us to work on your awesome online_actogram? :-) If yes, would you like to have admin rights on the forked repo (this doesn't mean you'll have to do anything, you're free to not contribute too, you already did a lot! :-) ).

**Comment by ryandesign** [comment]
> the project will be forked

Indeed it was forked and development was continued here: https://github.com/Circadiaware/webactogram

**Comment by lrq3000** [comment]
Yes it is, and although progress is currently slow, it is still being actively developed and maintained :-) thank you for linking to the fork!
